lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -738,3 +740,64 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
     upload_csv_to_report_store(output_rows, 'cohort_results', course_id, start_date)
 
     return task_progress.update_task_state(extra_meta=current_step)
+
+
+def push_ora2_responses_to_s3(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+    """
+    Collect ora2 responses and upload them to S3 as a CSV
+    """
+
+    start_time = datetime.now(UTC)
+
+    num_attempted = 1
+    num_succeeded = 0
+    num_failed = 0
+    num_total = 1
+    curr_step = "Collecting responses"
+
+    def update_task_progress():
```

Comment:
Can you explain why we need to do this instead of just using the `TaskProgress` object like in the other tasks?